go.mod

```diff
@@ -25,6 +25,7 @@ require (
 )
 
 require (
+	cloud.google.com/go/compute/metadata v0.2.3 // indirect
```

Comment:
I added this with:
```
go get cloud.google.com/go/compute/metadata
```
because `go mod tidy` would not resolve without it

Reply:
👍 I had noticed the issue, but doing `go get` on my end didn't seem to actually modify the `go.mod`, just made `go mod tidy` start working again.